src/torrent.cpp

```diff
@@ -1094,6 +1094,82 @@ namespace libtorrent
 #endif
 	}
 
+	boost::uint64_t torrent::get_flags() const
```

Comment:
I wonder if ``get_flags()`` and ``set_flags()`` should be modeled after http://en.cppreference.com/w/cpp/io/ios_base/flags